Code review for waa94/codingame-1.c

```diff
@@ -0,0 +1,136 @@
+// # Coding with reflexion
+
+// ## Power of Thor
+
+// _here his a link to CodinGame to use this solution :_
+// [CodinGame](https://www.codingame.com/)
+
+// **Here is the second version of my code following the review :** I have added
```

Comment:
Good, but remove this from your code -- it belong in the pull request discussion area.